### ISSUE TITLE
Am I missing any accessibility components?

### ISSUE THREAD

**Issue by emahadzimustafic** [emahadzimustafic/arrrgenes-treasure-tours#1]
Markbot says that my code is correct and I submitted it, but is there something that I missed? Can one of you check my website and/or code?

The screenshot provided in the folder is the exact same as the original website, so I'm not sure if I made the correct changes.

Thank you in advance.

@hellachella @carolsinc

**Comment by carolsinc** [comment]
If you were able to submit then you got all the necessary changes, but I can take a look for you

**Comment by emahadzimustafic** [comment]
Thank you, I appreciate it!

**Comment by carolsinc** [comment]
Everything looks good!
